cliApp.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM montagu-api-build-environment
+
```

Comment:
If I understand what you are doing here, this is the Dockerfile for the image that gets used to actually run the CLI in production. Whereas `cli.dockerfile` is the build environment image. In which case, this one should not extend from montagu-api-build-environment - we don't want our whole build env in production. You probably just want to extend from the same OpenJDK JRE image as the API `distDocker` task.